examples/reusing_functions/reusing_functions.ipynb

```diff
@@ -0,0 +1,1475 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Resusing Functions Example\n",
+    "\n",
+    "This notebook demonstrates the use of the subdag operator.\n",
+    "\n",
+    "The subdag operator allows you to effectively run a driver within a node.\n",
+    "In this case, we are calculating unique website visitors from the following set of parameters:\n",
+    "\n",
+    "1. Region = CA (canada) or US (United States)\n",
+    "2. Granularity of data = (day, week, month)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pre-requisites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You need to have the following installed and set up:\n",
+    "- Hamilton\n",
+    "- Pandas\n",
+    "\n",
+    "Execute the code commented below to install the above if you don't have them already installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %pip install sf-hamilton pandas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing all the things you need\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Any, Dict\n",
+    "import pandas as pd\n",
+    "import reusable_subdags\n",
```

Comment:
you don't need this import because the `%%cell_to_module` below will define `reusable_subdags`.